### PR TITLE
Full solution not being committed, bottom portion is being cropped out. #62(Solved)

### PR DESCRIPTION
**Pull request by mahbubcseju**
In some cases, it failed to commit full code. Please see the issue number #62 for details. I solved this issue by fetching the code from the individual submission details page.

### CODE REVIEW

File: scripts/leetcode.js [hunk 1]
```diff
@@ -210,6 +210,60 @@ function uploadGit(
...
+        for (var i = 0; i < scripts.length; i++) {
+          var text = scripts[i].innerText;
+          if(text.includes('pageData')) {
+            var firstIndex = text.indexOf('submissionCode');
+            var lastIndex = text.indexOf('editCodeUrl');
+            var sclicedText = text.slice(firstIndex, lastIndex);
+            var firstInverted = sclicedText.indexOf('\'');
+            var lastInverted = sclicedText.lastIndexOf('\'');
+            var codeUnicoded = sclicedText.slice(firstInverted + 1, lastInverted);
+            var code = codeUnicoded.replace(/\\u[\dA-F]{4}/gi,
```
Comment (QasimWani) [inline scripts/leetcode.js:239]: explain regex?

File: scripts/leetcode.js [hunk 2]
```diff
@@ -210,6 +210,60 @@ function uploadGit(
   });
 }
 
+function findCode(
```
Comment (QasimWani) [inline scripts/leetcode.js:213]: could you comment main parts of this function?
thanks

**Review by QasimWani — CHANGES_REQUESTED** [review]
thanks for the PR!
can you add some comments on your function? also, have you passed all eslint tests? We require you to lint before I can merge. keeps everything standardized. see readme for description.

**Comment by mahbubcseju** [comment]
I added some comments and ran lint test. Thanks

**Review by QasimWani — APPROVED** [review]
lgtm!
good job :D
also, next time, make sure to send PR to dev branch instead of main. @mahbubcseju

**Pull request merged**